stable/sonatype-nexus/templates/deployment.yaml

```diff
@@ -44,10 +44,10 @@ spec:
             - mountPath: /nexus-data
               name: nexus-data-volume
       volumes:
-      - name: nexus-data-volume
-        {{- if .Values.persistence.enabled }}
-        persistentVolumeClaim:
-          claimName: {{ template "fullname" . }}
-        {{- else }}
+        - name: nexus-data-volume
+          {{- if .Values.persistence.enabled }}
+          persistentVolumeClaim:
+            claimName: {{ template "fullname" . }}
+          {{- else }}
           emptyDir: {}
```

Comment:
`emptyDir` should be at the same indentation level as `name` and `persistentVolumeClaim`. This change fixes that.